modules/setting/repository.go

```diff
@@ -48,6 +48,7 @@ var (
 		DefaultBranch                           string
 		AllowAdoptionOfUnadoptedRepositories    bool
 		AllowDeleteOfUnadoptedRepositories      bool
+		DisabledDownloadSourceArchives          bool
```

Comment:
```suggestion
		DisableDownloadSourceArchives          bool
```

Reads better and is more consistent with the rest of the options here